Document/0x05e-Testing-Cryptography.md

```diff
@@ -57,6 +57,15 @@ Apps that target modern API levels, went through the following changes:
 - For Android 9 (API level 28) and above the [Android Developer Blog](https://android-developers.googleblog.com/2018/03/cryptography-changes-in-android-p.html "Cryptography Changes in Android P") shows even more aggressive changes:
   - You get a warning if you still specify a provider using the `getInstance` method and you target any API below 28. If you target Android 9 (API level 28) or above, you get an error.
   - The `Crypto` provider is now removed. Calling it will result in a `NoSuchProviderException`.
+- For Android 10 (API level 29) and the [Developer Documentation](https://developer.android.com/about/versions/10/behavior-changes-all#security "Security Changes in Android 10") shows that:
+  - The Conscrypt security provider includes a public API for TLS functionality.The collection of classes under android.net.ssl contain static methods to access functionality that isn't available from the generic javax.net.ssl APIs.
```

Comment:
As this is just copy and paste form the URL, please remove line 61 and following and jut reference to the Android documentation (see my suggestion above)